File: FindFirstAndLastPositionOfElementInSortedArray/FindFirstAndLastPositionOfElementInSortedArray.cpp
#include <bits/stdc++.h>
#include "solution.cpp"
using namespace std;

int main() {
    
    int n;
    cin >> n;
    vector<int> nums(n);
    for(int i = 0 ; i < n ; i++) {
        cin >> nums[i];
    }
    int q;
    cin >> q;
    for(int i = 0 ;i < q ; i++) {
        int target;
        cin >> target;
        vector<int> result = solution().searchRange(nums,target);
        cout << result[0] << " " << result[1] << "\n";
    }
    return 0;
}

File: FindFirstAndLastPositionOfElementInSortedArray/FindFirstAndLastPositionOfElementInSortedArray.py
from solution import solution

if __name__ == '__main__':
    n = int(input())
    nums = list(map(int, input().strip().split()))
    q = int(input())
    for _q in range(q):
        target = int(input())
        result = solution().searchRange(nums, target)
        print(*result)
